### PR TITLE
[ZH] Review and clean up miscellaneous translations

### PR DESCRIPTION
**Pull request by CloneWith**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Gameplay/Score/ScoreV1/osu!/zh.md [hunk 1]
```diff
@@ -18,22 +14,22 @@ no_native_review: true
...
+- *连击倍率*等于（这次打击前的连击数 - 1）或 0，两者取较大值。
 - *难度倍率*是该特定谱面所对应的一个值。[下文](#难度倍率)可见具体计算公式。
 - *模组倍率*是正在使用的模组组合起来的倍率。
 
 ### 难度倍率
 
 **难度倍率**等于正在玩的谱面用旧算法得出的星级评价。具体公式如下：
 
-`难度倍率 = Round((掉血速度 + 圆圈大小 + 整体难度 + Clamp (打击物件总数 / 以秒计的掉血速度 * 8, 0, 16)) / 38 * 5)`
+`难度倍率 = 四舍五入((掉血速度 + 圆圈大小 + 整体难度 + Clamp(打击物件总数 / 以秒计的掉血速度 * 8, 0, 16)) / 38 * 5)`
```
Comment (YumeMuzi) [inline wiki/Gameplay/Score/ScoreV1/osu!/zh.md:26]:
```suggestion
`难度倍率 = Round((掉血速度 + 圆圈大小 + 整体难度 + Clamp(打击物件总数 / 以秒计的掉血速度 * 8, 0, 16)) / 38 * 5)`
```
代码方法不可翻译。

**Review by YumeMuzi — CHANGES_REQUESTED** [review]

File: wiki/People/Nomination_Assessment_Team/SEV_rating/zh.md [hunk 2]
```diff
@@ -1,21 +1,17 @@
...
-*注意：如果为了应用谱面未在过审之前提交的修改内容，而重置了谱面的提名，这个分数始终为 0/0。这套系统并不是用于阻止摸和修改并提升[过审 (Qualified)](/wiki/Beatmap/Category#过审-(qualified)) 谱面的。*
+*注意：如果为了应用谱面未在过审之前提交的修改内容，而重置了谱面的提名，这个分数始终为 0/0。这套系统并不是用于阻止摸和修改并改善[过审 (Qualified)](/wiki/Beatmap/Category#过审-(qualified)) 谱面的。*
 
 **明显性**与问题被发现的容易程度有关。
 
 | 分数 | 定义 | 解释 |
 | :-: | :-- | :-- |
 | 0 | 不明显 | 如果问题难以察觉或问题不大，不需要持续跟进解决，则适用此条。 |
-| 1 | 有经验的人可以看出 | 需要一定的知识、经验、注意力来发现。通常工具或普通的玩家都无法察觉，例如时间轴 (Timing) 问题和歌曲元数据 (Metadata) 问题。 |
+| 1 | 有经验的人可以看出 | 需要一定的知识/经验/注意力来发现。工具或普通的玩家通常都无法察觉，例如测定时间轴 (Timing) 问题和歌曲元数据问题。 |
```
[the remark this answers is not in the document]
Comment (CloneWith) [reply wiki/People/Nomination_Assessment_Team/SEV_rating/zh.md:14]: 1. `知识...注意力` 是按照原文意思更改(?)
2. `通常` 的语序调换，可能会比较好
3. `测定时间轴` 按照制图过程的名词更改
4. `元数据` 认为可以不翻译出英文原文

以上四点修改，个人觉得还是有一些争议

**Review by YumeMuzi — APPROVED** [review]

**Review by Walavouchey — CHANGES_REQUESTED** [review]
some "，" are changed to "、" and vice versa. which one should it be?

**Comment by CloneWith** [comment]
> some "，" are changed to "、" and vice versa. which one should it be?

We prefer to use `、` when it contains the meaning of "and / or". So I have modified most of them to match the structure of the sentence. And the remaining ones are kept intentionally.

**Review by Walavouchey — APPROVED** [review]
i'm losing my mind. why are there two commas in chinese.

**Pull request merged**